lib/heroku/cli.rb

```diff
@@ -1,3 +1,8 @@
+if RUBY_VERSION < '1.9.0'
+  $stderr.puts "WARNING: Heroku Toolbelt will require Ruby 1.9+ beginning next week"
+  $stderr.puts "https://github.com/heroku/heroku/pull/1479"
```

Comment:
Awesome. And for the record, I think it was the right call to leave the date out. I was mostly wondering how it'd look like after the next week.